bin/tessel-2.js

```diff
@@ -40,6 +40,11 @@ function callControllerCallback(methodName) {
   };
 }
 
+parser.command('install-drivers')
+  .callback(function() {
+    require('./tessel-install-drivers');
```

Comment:
It's considered a very bad practice to run logic just by including a file. Please export a function, move the require to the top of the file, and call the function here :)